specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-15-preview/machineLearningServices.json

```
@@ -0,0 +1,5532 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ComputeNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation was successful. The response contains the list of IP addresses.",
+            "schema": {
+              "$ref": "#/definitions/AmlComputeNodesInformation"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get compute nodes information for a compute": {
+            "$ref": "./examples/amlComputeListNodes.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/listKeys": {
+      "post": {
+        "tags": [
+          "OperationalizationClusters",
+          "MachineLearningComputes"
+        ],
+        "operationId": "MachineLearningCompute_ListKeys",
+        "description": "Gets secrets related to Machine Learning compute (storage keys, service credentials, etc).",
+        "x-ms-examples": {
+          "List AKS Compute Keys": {
+            "$ref": "./examples/listKeysAKSCompute.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ComputeNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "schema": {
+              "$ref": "#/definitions/ComputeSecrets"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/workspaces/skus": {
+      "get": {
+        "tags": [
+          "WorkspaceSkus"
+        ],
+        "description": "Lists all skus with associated features",
+        "operationId": "ListSkus",
```

Maybe this makes more sense based on the tags on this operation?
```suggestion
        "operationId": "Workspaces_ListSkus",
```

---

same this, out of scope - this is the same as 06-01